### ISSUE TITLE
Kreia runs a Weequay fight club on her time off

### ISSUE THREAD

**Issue by JCarter426** [KOTORCommunityPatches/TSL_Community_Patch#22]
(reported by Chonch)

![image](https://user-images.githubusercontent.com/46898647/61348594-3f461e80-a82f-11e9-9940-bef4a977ff23.png)

Bizarre bug of Kreia fighting a Weequay on the fake _Ebon Hawk_ while telepathically talking to the Exile about the echo on Nar Shaddaa. Sniggles said it's happened before too, so the trick is figuring out how to reproduce it.

My only guess is maybe an Exchange thug from one of the random encounters teleports over to her somehow.

**Comment by DarthParametric** [comment]
Yeah I've heard of this before but never seen it firsthand, I seem to recall I looked at her little fake room off to the side and couldn't spot an obvious way hostiles could get there, but perhaps the walkmesh warrants closer examination.

**Comment by DarthParametric** [comment]
I had a look at the walkmesh for the room and there doesn't appear to be any noticeable gaps or other geometry issues. There are also no room links from/to that room.

**Comment by DarthParametric** [comment]
Perhaps the brute force solution is to place a trigger over the entire hidden room and have it check the faction of any entering object. If they are hostile, destroy them. Not ideal, but it should at least prevent the conversation interruption, even if we can't determine the specific means by which mooks are teleporting to her.
